src/main/java/core/basesyntax/service/ReadFromFileService.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class ReadFromFileService {
+    private StringBuilder resultBuilder = new StringBuilder();
```

Comment:
why is this on the class level? what if you call `readFromFile()` multiple times?